samtools-1.3/Makefile

```diff
@@ -0,0 +1,22 @@
+build_tool = runtime-container.DONE
+git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../samtools-1.3 | cut -f1 -d " ")
+name = quay.io/ucsc_cgl/samtools
```

Comment:
@hannes-ucsc when in a separate directory, should the version also be in the name?